src/screens/FeedScreen/ContentLink.js

```diff
@@ -57,7 +58,7 @@ const ContentLink = ({
   };
 
   return (
-    <View style={styles.contentFeed}>
+    <View style={styles.contentFeed(hasComment)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `hasComment` prop is used to conditionally adjust the style of the content feed. However, it's not clear what happens when `hasComment` is `undefined`. It would be better to ensure that `hasComment` always has a boolean value.
```diff
-    <View style={styles.contentFeed(hasComment)}>
+    <View style={styles.contentFeed(!!hasComment)}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->